src/wallet/rpcwallet.cpp

```diff
@@ -2152,6 +2153,7 @@ static RPCHelpMan lockunspent()
                             },
                         },
                     },
+                    {"store_lock", RPCArg::Type::BOOL, RPCArg::Default{false}, "Whether to write/erase this lock in the wallet database, or keep the change in memory only."},
```

Comment:
I would slightly prefer for this parameter to be called "persistent". I think it better captures the meaning.

Reply:
Sure, done